iotilecore/iotile/core/hw/virtual/virtualtile.py

```diff
@@ -38,7 +38,7 @@ def _check_convert_name(cls, name):
         if not isinstance(name, bytes):
             name = name.encode('utf-8')
         if len(name) < 6:
-            name += bytes(' ')*(6 - len(name))
+            name += bytes(' '.encode('utf-8'))*(6 - len(name))
```

Comment:
Also I think this could be rewritten as simply `b' '*(6 - len(name))` as well

Reply:
Good point. Simplified.